src/core/dfn-map.js

```diff
@@ -8,10 +8,10 @@ export const definitionMap = Object.create(null);
  * @param {string[]} names Names to register the element by
  */
 export function registerDefinition(dfn, names) {
-  for (const name of names) {
-    if (!definitionMap[name]) {
+  for (const name of names.map(name => name.toLowerCase())) {
+    if (name in definitionMap === false) {
       definitionMap[name] = [dfn];
-    } else if (!definitionMap[name].includes(dfn)) {
+    } else if (definitionMap[name].includes(dfn) === false) {
```

Comment:
I understand the desire to use `===` in `in` expression as it can remove parentheses, but with `includes()`? Not sure...

Reply:
I added the `false`s for (my own) legibility. As this code runs, each of the checks are somewhat complex and having the explicit `false` was just to helped me with cognitive load. I'm ok to remove them tho.